language/en-GB/mod_articles.ini

```
@@ -0,0 +1,100 @@
+; Joomla! Project
+; (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+MOD_ARTICLES="Articles"
+MOD_ARTICLES_FIELD_ARTICLEGROUPINGDIR_LABEL="Grouping Direction"
+MOD_ARTICLES_FIELD_ARTICLEGROUPING_LABEL="Article Grouping"
+MOD_ARTICLES_FIELD_ARTICLEORDERINGDIR_LABEL="Ordering Direction"
+MOD_ARTICLES_FIELD_ARTICLEORDERING_LABEL="Article Field to Order By"
+MOD_ARTICLES_FIELD_AUTHORALIASFILTERING_LABEL="Author Alias Filtering Type"
+MOD_ARTICLES_FIELD_AUTHORALIAS_LABEL="Author Aliases"
+MOD_ARTICLES_FIELD_AUTHORFILTERING_LABEL="Author Filtering Type"
+MOD_ARTICLES_FIELD_AUTHOR_LABEL="Authors"
+MOD_ARTICLES_FIELD_CATDEPTH_LABEL="Category Depth"
+MOD_ARTICLES_FIELD_CATFILTERINGTYPE_DESC="Include or exclude the selected categories."
+MOD_ARTICLES_FIELD_CATFILTERINGTYPE_LABEL="Category Filtering Type"
+MOD_ARTICLES_FIELD_COUNT_DESC="The number of items to display. The default value of 0 will display all articles."
+MOD_ARTICLES_FIELD_COUNT_LABEL="Articles to Display"
+MOD_ARTICLES_FIELD_DATEFIELDFORMAT_DESC="Please enter in a valid date format. See: https://php.net/date for formatting information."
+MOD_ARTICLES_FIELD_DATEFIELDFORMAT_LABEL="Date Format"
+MOD_ARTICLES_FIELD_DATEFIELD_LABEL="Date Field"
+MOD_ARTICLES_FIELD_DATEFILTERING_LABEL="Date Filtering"
+MOD_ARTICLES_FIELD_DATEGROUPINGFIELD_DESC="Select which date field you want the date grouping to be applied to."
+MOD_ARTICLES_FIELD_DATEGROUPINGFIELD_LABEL="Date Grouping Field"
+MOD_ARTICLES_FIELD_DATERANGEFIELD_LABEL="Date Range Field"
+MOD_ARTICLES_FIELD_ENDDATE_LABEL="To Date"
+MOD_ARTICLES_FIELD_EXCLUDEDARTICLES_LABEL="Article IDs to Exclude"
+MOD_ARTICLES_FIELD_EXCLUDE_CURRENT_LABEL="Exclude Current Article"
+MOD_ARTICLES_FIELD_GROUP_DISPLAY_LABEL="Display Options"
+MOD_ARTICLES_FIELD_GROUP_FILTERING_LABEL="Filtering Options"
+MOD_ARTICLES_FIELD_GROUP_GROUPING_LABEL="Grouping Options"
+MOD_ARTICLES_FIELD_GROUP_ORDERING_LABEL="Ordering Options"
+MOD_ARTICLES_FIELD_IMAGES_ARTICLE_LABEL="Show Intro/Full Image"
+MOD_ARTICLES_FIELD_IMAGES_DESC="Show the images that are inside the text of the article."
+MOD_ARTICLES_FIELD_IMAGES_LABEL="Show Article Images"
+MOD_ARTICLES_FIELD_INCLUDEDARTICLES_LABEL="Article IDs to Include"
+MOD_ARTICLES_FIELD_INFOLAYOUT_DESC="Layout for the Articles Info (Date, Hits, Category, Author)."
+MOD_ARTICLES_FIELD_INFOLAYOUT_LABEL="Articles Info Layout"
+MOD_ARTICLES_FIELD_INTROTEXTLIMIT_DESC="Truncate Introtext to a specified number of characters. Set to 0 to disable truncation."
+MOD_ARTICLES_FIELD_INTROTEXTLIMIT_LABEL="Introtext Limit (characters)"
+MOD_ARTICLES_FIELD_LAYOUT_COL_LABEL="Max. Number of Columns"
+MOD_ARTICLES_FIELD_LAYOUT_DESC="Display articles vertical or horizontal."
```

The correct english for this would be "Display articles vertically or horizontally."

But I think it _might_ be simpler english to write
```suggestion
MOD_ARTICLES_FIELD_LAYOUT_DESC="Display articles vertical or horizontal."
```

```suggestion
MOD_ARTICLES_FIELD_LAYOUT_DESC="Vertical or horizontal display of articles."
```